pubtools/_impl/pluggy.py

```diff
@@ -94,4 +94,22 @@ def task_context():
         pm.hook.task_stop(failed=failed)
 
 
+@hookspec(firstresult=True)
+def get_cert_key_paths_plugin(server_url):
+    """Get location of SSL certificates for a given service.
```

Comment:
Minor: "for a given service" is vague - it could mean for example that this is meant to be called server-side.

"used to authenticate with a given service" may be clearer.

Reply:
Reworded.